.ci/visual-regression-test.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Variables
+BUILD_DIR=$(pwd)
+GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Check if we are NOT on the master branch and this is a PR
+if [[ ${CURRENT_BRANCH} != "master" && -z ${CI_PR_URL} ]];
+then
+  echo -e "\nVisual regression tests will only run if we are not on the master branch and making a pull request"
+  exit 0;
+fi
+
+echo -e "\nProcessing pull request #$PR_NUMBER"
+
+LAST_GIT_COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+
+GIT_FILE_MODIFIED()
+{
+    # Stash list of changed files
+    GIT_FILES_CHANGED="$(git diff master --name-only)"
+
+    while read -r changedFile; do
+        if [[ "${changedFile}" == "$1" ]]
+        then
+            return 0;
+        fi
+    done <<< "$GIT_FILES_CHANGED"
+
+    return 1;
+}
+
+# Always run visual tests if "visual regression test" is in the last commit message
+if [[ ${LAST_GIT_COMMIT_MESSAGE} != *"visual regression test"* ]]
+then
+
+    # Skip visual tests if there hasn't been a modification to composer.lock
+    if ! GIT_FILE_MODIFIED 'composer.lock'
+    then
+        echo -e "\nSkipping visual regression tests since composer.lock has NOT changed"
+        exit 0
+    fi
+
+    # Skip visual tests if has been a modification to composer.json
+    if GIT_FILE_MODIFIED 'composer.json'
+    then
+        echo -e "\nSkipping visual regression tests since composer.json HAS changed"
+        exit 0
+    fi
+
+else
+    echo -e "\nRunning visual regression tests because the latest commit message demands it"
+fi
+
+# Stash site URLs
+MULTIDEV_SITE_URL="https://$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io/"
+LIVE_SITE_URL="https://live-$TERMINUS_SITE.pantheonsite.io/"
+
+# Ping the multidev environment to wake it from sleep
+echo -e "\nPinging the ${TERMINUS_ENV} multidev environment to wake it from sleep..."
+curl -I "$MULTIDEV_SITE_URL" >/dev/null
+
+# Ping the live environment to wake it from sleep
+echo -e "\nPinging the live environment to wake it from sleep..."
+curl -I "$LIVE_SITE_URL" >/dev/null
+
+# Check for custom backstop.json
```

Comment:
Maybe package this up as `terminus visual-regression:backstop:init`

Reply:
With this code in php rather than bash, `jq` becomes unnecessary

Reply:
See [my comment above](https://github.com/pantheon-systems/example-drops-8-composer/pull/197#discussion_r269584572) about creating new Terminus commands.

We could perhaps remove the `jq` dependency and do the replacement with `sed` or something else already available.